base/ref.go

```diff
@@ -0,0 +1,30 @@
+package base
+
+import (
+	"fmt"
+
+	"github.com/qri-io/qri/repo"
+)
+
+// InLocalNamespace checks if a dataset ref is local, assumes the reference is already canonicalized
+func InLocalNamespace(r repo.Repo, ref *repo.DatasetRef) bool {
+	p, err := r.Profile()
+	if err != nil {
+		return false
+	}
+
+	return p.ID == ref.ProfileID
+}
+
+// SetPublishStatus updates the Published field of a dataset ref
+func SetPublishStatus(r repo.Repo, ref *repo.DatasetRef) error {
```

Comment:
This comment doesn't appear to match what the function body is doing. Nothing here is modifying the Publish field. Is the expectation that the caller sets the field and then calls this to serialize the value? Rather, I would expect that this would take a bool parameter and assign it to the Published field.